doc/features.md

```diff
@@ -127,3 +127,22 @@ Beware that `CONFIG` section contents must comply with YAML syntax restrictions:
 * `- "silex/silex: *"` without quotes is an invalid YAML.
 * `- "silex/silex: ~1.2"` without quotes is a YAML object and refused by melody.
 * `- "-S"` without quotes is an array of arrays.
+
+
+Using fork and private repositories
+-----------------------------------
+
+If you need to use packages not registred in Packagist repository, you can specify repositories
+in the YAML configuration. See Composer documentation https://getcomposer.org/doc/04-schema.md#repositories
```

Comment:
I think that `https://getcomposer.org/doc/05-repositories.md` is more appropriate.
Wrap your link with a label: `See [composer documention](https://getcomposer.org/doc/05-repositories.md)`